### ISSUE TITLE
failed to inference

### ISSUE THREAD

**Issue by Hsintao** [traveller59/torch2trt#10]
environment:
`tensorrt:6.0.1.5
cuda:10.0
cudnn:7.6.0`
i use the serialized resnet50.onnx which from the torchvision.models.resnet50.

eror log
`[TensorRT] WARNING: TensorRT was linked against cuDNN 7.6.3 but loaded cuDNN 7.6.0
[TensorRT] WARNING: TensorRT was linked against cuDNN 7.6.3 but loaded cuDNN 7.6.0
[TensorRT] ERROR: ../rtSafe/cuda/cudaPoolingRunner.cpp (84) - Cudnn Error in execute: 8 (CUDNN_STATUS_EXECUTION_FAILED)
[TensorRT] ERROR: FAILED_EXECUTION: std::exception
`
can u give me some advice? thanks

**Comment by WangXuanBT** [comment]
i'm sorry, but,did you find the reason? i faced a similar issue:
[E] [TRT] ../rtSafe/cuda/cudaPoolingRunner.cpp (200) - Cudnn Error in execute: 3 (CUDNN_STATUS_BAD_PARAM)

**Comment by wxinbeings** [comment]
> i'm sorry, but,did you find the reason? i faced a similar issue: [E] [TRT] ../rtSafe/cuda/cudaPoolingRunner.cpp (200) - Cudnn Error in execute: 3 (CUDNN_STATUS_BAD_PARAM)

I got the same error, have you solved it? Could you give me some advice?Thanks.

**Comment by WangXuanBT** [comment]
> > i'm sorry, but,did you find the reason? i faced a similar issue: [E] [TRT] ../rtSafe/cuda/cudaPoolingRunner.cpp (200) - Cudnn Error in execute: 3 (CUDNN_STATUS_BAD_PARAM)
> 
> I got the same error, have you solved it? Could you give me some advice?Thanks.

Yeah, I'll show u my step(I test a craft.onnx in my case):
Trying process:
./trtexec --verbose --onnx=/tmp/trt/craft_modify_fix.onnx
**Success**

./trtexec --verbose --onnx=/tmp/trt/craft_modify_dynamic.onnx
_[E] [TRT] Network has dynamic or shape inputs, but no optimization profile has been defined._

./trtexec --verbose --onnx=/tmp/trt/craft_modify_dynamic.onnx --explicitBatch
_[E] [TRT] ../rtSafe/cuda/cudaPoolingRunner.cpp (200) - Cudnn Error in execute: 3 (CUDNN_STATUS_BAD_PARAM)_
then u should refer:
https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#work_dynamic_shapes
https://github.com/NVIDIA/TensorRT/tree/f5e8b8c55060447c4d3a45e7379bc25374edc93f/samples/opensource/trtexec
to learn something

./trtexec --verbose --onnx=/tmp/trt/craft_modify_dynamic.onnx --explicitBatch --device=1 --dumpOutput --dumpProfile  --useCudaGraph --workspace=2048 --minShapes=input:1x3x200x200 --optShapes=input:1x3x500x500 --maxShapes=input:1x3x800x800 --shapes=input:1x3x500x500
_[E] [TRT] ../rtSafe/cuda/cudaConvolutionRunner.cpp (362) - Cudnn Error in execute: 9 (CUDNN_STATUS_NOT_SUPPORTED)_

./trtexec --verbose --onnx=/tmp/trt/craft_modify_dynamic.onnx --explicitBatch --device=1 --dumpOutput --dumpProfile  --useCudaGraph --workspace=2048 --minShapes=input:1x3x200x200 --optShapes=input:1x3x608x608 --maxShapes=input:1x3x800x800 --shapes=input:1x3x500x500
_[E] [TRT] Concat_150: dimensions not compatible for concatenation_

./trtexec --verbose --onnx=/tmp/trt/craft_modify_dynamic.onnx --explicitBatch --device=1 --dumpOutput --dumpProfile  --useCudaGraph --workspace=2048 --minShapes=input:1x3x200x200 --optShapes=input:1x3x608x608 --maxShapes=input:1x3x800x800 --shapes=input:1x3x608x608
_[E] [TRT] ../rtSafe/safeContext.cpp (133) - Cudnn Error in configure: 7 (CUDNN_STATUS_MAPPING_ERROR)_

./trtexec --verbose --onnx=/tmp/trt/craft_modify_dynamic.onnx --explicitBatch --dumpOutput --dumpProfile  --useCudaGraph --workspace=2048 --minShapes=input:1x3x200x200 --optShapes=input:1x3x608x608 --maxShapes=input:1x3x800x800 --shapes=input:1x3x608x608
**Success**

In conclusion, u have to learn the useage from https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#work_dynamic_shapes and https://github.com/NVIDIA/TensorRT/tree/f5e8b8c55060447c4d3a45e7379bc25374edc93f/samples/opensource/trtexec and so on...
